TTT.cpp

```diff
@@ -0,0 +1,6 @@
+#include "TTT.h"
+
+TTT:TTT()
+{
+
+}
```

Comment:
I think it's a good convention to terminate every source file with a newline. GitHub keeps yelling at us for this, and AFAIK most C++ style guides have a spec for this.